api/doc/thor.yaml

```diff
@@ -12,7 +12,7 @@ info:
   license:
     name: LGPL 3.0
     url: https://www.gnu.org/licenses/lgpl-3.0.en.html
-  version: 2.1.2
```

Comment:
Leave this with the PR for API changes, here it should be #776 ,wdyt?

Reply:
Ahh, forgot I did that there, will close